packages/settings-library/src/settings_library/node_ports.py

```diff
@@ -0,0 +1,45 @@
+import json
+from datetime import timedelta
+from typing import Any, Final
+
+from pydantic import Field, NonNegativeInt, PositiveInt, SecretStr
+from pydantic.json import pydantic_encoder
+
+from .base import BaseCustomSettings
+from .postgres import PostgresSettings
+from .storage import StorageSettings
+
+NODE_PORTS_400_REQUEST_TIMEOUT_ATTEMPTS_DEFAULT_VALUE: Final[NonNegativeInt] = 3
+
+
+class StorageAuthSettings(StorageSettings):
+    STORAGE_LOGIN: str
```

Comment:
We use ```STORAGE_USERNAME``` USERNAME all over the place. maybe let's keep it like so?

Reply:
This is derived form basicauth, but we can rename it for constancy.